M/mdbtools/build_tarballs.jl

```diff
@@ -0,0 +1,62 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "mdbtools"
+version = v"0.8.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/cyberemissary/mdbtools.git", "b753ff36a0f1d88ae8a300ed6712f4aa2ddb7d08")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd mdbtools/
+autoreconf -if
+apk add glib-dev
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-man
+make -j8
```

Comment:
```suggestion
make -j${nproc}
```

Reply:
Resolved in https://github.com/JuliaPackaging/Yggdrasil/pull/1322/commits/ff2dc197095166197d8e8b3e043190ad7ea7c905.